### PR TITLE
feat(ui): sending indicator in BottomRow

### PR DESCRIPTION
**Pull request by geweald**
# Submit a pull request

## CLA

- [X] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

## Description of the pull request
Continuation of #1349 

Not sure tho if I should remove all the builders from the message widget that are used in the bottom row? As people can now add them with `copyWith`

### CODE REVIEW

File: packages/stream_chat_flutter/lib/src/message_widget/bottom_row.dart [hunk 1]
```diff
@@ -88,6 +89,61 @@ class BottomRow extends StatelessWidget {
   /// {@macro usernameBuilder}
   final Widget Function(BuildContext, Message)? usernameBuilder;
 
+  /// {@macro sendingIndicatorBuilder}
```
Comment (HayesGordon) [inline packages/stream_chat_flutter/lib/src/message_widget/bottom_row.dart:92]: Does the template for `sendingIndicatorBuilder` exist?

**Comment by HayesGordon** [comment]
@geweald sorry about that. That is my mistake for missing the old PR and the discussion there.

**Comment by xsahil03x** [comment]
Hey @geweald , I created a PR on your branch to make it non-breaking. Can you test and merge that so that we can prepare to release? Thanks 🙏🏼

**Comment by xsahil03x** [comment]
Hey @geweald, I am continuing the changes in a separate PR as we are looking to release today.
https://github.com/GetStream/stream-chat-flutter/pull/1416

**Pull request closed without merging**